docs/en/basechain-release-notes.md

```diff
@@ -8,6 +8,54 @@ sidebar_label: Basechain
 
 Reverse chronology order of release notes.
 
+## Pre-Basechain Build 1346 Hard Fork - 2019/11/26
+
+**Breaking changes:**
+- The `eth_getTransactionCount` and eth_call Web3 JSON-RPC methods on the `/eth` endpoint now expect
+  the input address to be an Ethereum account, not a Loom account.
+- `eth_getLogs` and any other Web3 JSON-RPC methods that query a range of blocks will now only allow
+  querying up to 20 blocks per request on public Extdev nodes.
+
+**New features:**
+- A node will now panic when it’s started using a jump-start that requires a newer build than the
+  node is currently running, once the build is upgraded the node can simply be restarted without
+  restoring the working directory from the jump-start. This requires a jump-start that’s created
+  after the `chaincfg:v1.4` feature flag is enabled, which means it will only work with future
+  jump-starts (the first of which will be available a few days after this release).
+- The `eth_sendRawTransaction` JSON-RPC method on the `/eth` endpoint now works similarly to the way
+  one would expect it to work on an Ethereum node. Note that an account mapping between the Ethereum
+  account that’s used to sign the Ethereum tx and an Extdev account must exist before `eth_sendRawTransaction`
```

Comment:
```suggestion
  account that’s used to sign the Ethereum tx and a Loom account must exist before `eth_sendRawTransaction`
```

Reply:
`Loom` -> `Basechain`

Reply:
> `Loom` -> `Basechain`

I suggested Loom because this applies to both Basechain and Extdev. But maybe Basechain is better.